lib/galaxy/webapps/galaxy/api/lda_datasets.py

```diff
@@ -404,19 +404,19 @@ def load( self, trans, payload=None, **kwd ):
         """
         if payload:
             kwd.update(payload)
-        kwd['space_to_tab'] = False
-        kwd['to_posix_lines'] = True
+        kwd[ 'space_to_tab' ] = False
+        kwd[ 'to_posix_lines' ] = True
         kwd[ 'dbkey' ] = kwd.get( 'dbkey', '?' )
         kwd[ 'file_type' ] = kwd.get( 'file_type', 'auto' )
-        kwd['link_data_only'] = 'link_to_files' if util.string_as_bool( kwd.get( 'link_data', False ) ) else 'copy_files'
+        kwd[ 'link_data_only' ] = 'link_to_files' if util.asbool( kwd.get( 'link_data', False ) ) else 'copy_files'
```

Comment:
Is this needed to fix some bug? `asbool()` already accepts 'true', 'yes', 'on', '1' (not case sensitive) as `True`.